assets/css/index.css

```diff
@@ -1,4 +1,63 @@
 /* added to check the style sheet was linking */
+
+header {
+    background: linear-gradient( rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5) ), url('https://slack-imgs.com/?c=1&url=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1520881363902-a0ff4e722963%3Fixlib%3Drb-1.2.1%26q%3D80%26fm%3Djpg%26crop%3Dentropy%26cs%3Dtinysrgb%26w%3D1080%26fit%3Dmax');
+    height: 100vh;
+    background-repeat: no-repeat;
+    background-size: 100%;
+}
+
 body {
-    background-color: purple;
+    margin: 0;
+}
+
+h1 {
+    margin: 0;
+    font-family: 'Montserrat', sans-serif;
+    font-size: 50px;
+}
+
+hr {
```

Comment:
It's very subtle, but on my screen the `<hr>` element is somehow appearing with _two_ colours. You'll really have to zoom in to see it.

I noticed that adding the below CSS rule makes it look a little cleaner.
```css
hr {
  ...
  border: 1px solid white;
}
```

Before | After
--- | ---
![image](https://user-images.githubusercontent.com/13058213/50687632-01fc0b00-101a-11e9-9210-4e7c3395b29d.png) | ![image](https://user-images.githubusercontent.com/13058213/50687655-1213ea80-101a-11e9-8669-138bb3d91567.png)

Reply:
I checked this and noticed the same thing. Amends made in my latest commit 4fa9a60bb164500bbf596c8f2ce2ee2e6638b2c7.